### PR TITLE
CPA-91 link new poem with single poem

### PR DESCRIPTION
**Pull request by RudeMax**
Now after creating a new poem, it is opened in a single poem window. 
BTW!!! The response from the **backend has to be improved**, when I create a new poem **it returns tags not in a form of array but a string like that: `tags: "bla, blabla, smth"`**. Kostia has suggested to add a checker and converter to Array in Poem.js and it turned out to solve the problem, but when fix the backend it can be removed.

### CODE REVIEW

File: src/redux/poems/poemsReducer.js [hunk 1]
```diff
@@ -52,10 +53,12 @@ const poemsReducer = (state = initalState, action) => {
 				isPostingPoem: true,
 			};
 		case CREATE_POEM_SUCCESS:
+			console.log(action.payload)
```
Comment (KenAdams777) [inline src/redux/poems/poemsReducer.js:56]: Remove `console.log`, please.

File: src/redux/poems/poemsReducer.js [hunk 2]
```diff
@@ -52,10 +53,12 @@ const poemsReducer = (state = initalState, action) => {
...
 			return {
 				...state,
 				isPostingPoem: false,
 				isPostedPoem: true,
+				currentPoem: { ...action.payload }
```
Comment (KenAdams777) [inline src/redux/poems/poemsReducer.js:61]: Just curious what is the reason for destruction here?

**Review by KenAdams777 — APPROVED** [review]

**Review by KenAdams777 — APPROVED** [review]
Also, forgot to add suggestion to make the `Add poem` `button` `disabled` if a user is not `Logged`.

**Review by bejalane — APPROVED** [review]

File: src/redux/poems/poemsReducer.js (hunk 2 above)
Comment (RudeMax) [reply src/redux/poems/poemsReducer.js:61]: Long story to tell, will remove it)

**Pull request merged**